models/Position.js

```diff
@@ -20,6 +28,13 @@ const Position = sequelize.define('position', {
             notEmpty: { msg: 'Position name should be set.' },
         }
     },
+    body_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
```

Comment:
actually it can be nullable, I propose to remove the validate block and then set allowNull: true